app/main.cpp

```diff
@@ -153,9 +179,6 @@ static void init_qgis(const QString& qgisDataPath)
 #ifdef ANDROID
   // QGIS plugins on Android are in the same path as other libraries
   QgsApplication::setPluginPath( QApplication::applicationDirPath() );
-  QgsApplication::setPkgDataPath(qgisDataPath);
```

Comment:
we don't need to set the data path? is it already set to the correct directory?

Reply:
PkgDataPath is set by default to:` /data/user/0/uk.co.lutraconsulting/files/share`
Since srs.db is copied there and by default read from `pkgDataPath() + QStringLiteral( "/resources/srs.db" ); ` or from tempFolder, where is copied afterwards from pkgDataPath.

Reply:
Eventually pkgData default value is overwritten since there is losing permission issue when copying resources files to system pkg path.